app/_src/guides/builtin-gateway.md

```diff
@@ -0,0 +1,340 @@
+apiVersion: kuma.io/v1alpha1
+kind: MeshHTTPRoute
+metadata:
+ name: edge-gateway-route
+ namespace: {{site.mesh_namespace}} 
+ labels:
+   kuma.io/mesh: default
+spec:
+ targetRef:
+   kind: MeshGateway
+   name: my-gateway
+ to:
+ - targetRef:
+     kind: Mesh
+   rules:
+   - matches:
+     - path:
+         type: PathPrefix
+         value: "/"
+     default:
+       backendRefs:
+       - kind: MeshService
+         name: demo-app_kuma-demo_svc_5000
+" | kubectl apply -f -
+```
+
+Now if we try to reach our gateway again: 
+```shell
+$ curl -v ${PROXY_IP}:8080
+*   Trying 127.0.0.1:8080...
+* Connected to 127.0.0.1 (127.0.0.1) port 8080
+> GET / HTTP/1.1
+> Host: 127.0.0.1:8080
+> User-Agent: curl/8.4.0
+> Accept: */*
+>
+< HTTP/1.1 403 Forbidden
+< content-length: 19
+< content-type: text/plain
+< date: Fri, 09 Feb 2024 10:10:16 GMT
+< server: Kuma Gateway
+< x-envoy-upstream-service-time: 24
+<
+* Connection #0 to host 127.0.0.1 left intact
+RBAC: access denied%
+```
+
+We see we're getting a forbidden error. This is because we do have permissions from the gateway to talk to our demo-app service.
+Let's add a [`MeshTrafficPermission`](/docs/{{ page.version }}/policies/meshtrafficpermission):
+
+```shell
+echo "
+apiVersion: kuma.io/v1alpha1
+kind: MeshTrafficPermission
+metadata:
+  namespace: {{ site.mesh_namespace }} 
+  name: demo-app
+spec:
+  targetRef:
+    kind: MeshService
+    name: demo-app_kuma-demo_svc_5000
+  from:
+    - targetRef:
+        kind: MeshService
+        name: edge-gateway 
+      default:
+        action: Allow
+" | kubectl apply -f -
+```
+
+You can check it works with:
+
+```shell
+curl -XPOST -v ${PROXY_IP}:8080/increment
+*   Trying 127.0.0.1:8080...
+* Connected to 127.0.0.1 (127.0.0.1) port 8080
+> POST /increment HTTP/1.1
+> Host: 127.0.0.1:8080
+> User-Agent: curl/8.4.0
+> Accept: */*
+>
+< HTTP/1.1 200 OK
+< x-powered-by: Express
+< content-type: application/json; charset=utf-8
+< content-length: 42
+< etag: W/"2a-gDIArbqhTz783Hls/ysnTwRRsmQ"
+< date: Fri, 09 Feb 2024 10:24:33 GMT
+< x-envoy-upstream-service-time: 6
+< server: Kuma Gateway
+<
+* Connection #0 to host 127.0.0.1 left intact
+{"counter":3265,"zone":"local","err":null}
+```
+
+See how now the demo-app responds.
+
+## Securing your public endpoint with a certificate
+
+We've exposed our application to a public endpoint thanks to our gateway.
+However, we probably now want to add TLS to our endpoint.
+
+### Create a certificate
+
+For this demo we'll create a self-signed certificate:
+
+```shell
+openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout tls.key -out tls.crt -subj "/CN=${PROXY_IP}"
+```
+
+
+```shell
+echo "
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-gateway-certificate
+  namespace: {{ site.mesh_namespace }} 
+  labels:
+    kuma.io/mesh: default
+data:
+  value: "$(cat tls.key tls.cert | base64)"
+type: system.kuma.io/secret
+" | kubectl apply -f - 
+```
+
+Now let's update our gateway to use this certificate
```

Comment:
**Suggestion** (nit): staying consistent with the rest of the "instruction plus command block" parts.

```suggestion
Now let's update our gateway to use this certificate:
```